packages/queries/src/schema/isEnumColumn.ts

```diff
@@ -0,0 +1,9 @@
+import { ColumnDef } from '../types/Schema';
+
+export function isEnumColumn(columnDef: ColumnDef): boolean {
+    if (columnDef.properties.type.enum) {
```

Comment:
no need for an if statement. Just `return columnDef.properties.type.enum` is enough

Reply:
Enum is possibly `undefined`